rgsync/Connectors/sql_connectors.py

```diff
@@ -206,12 +205,11 @@ def __init__(self, connection, tableName, pk, exactlyOnceTableName=None):
 
     def PrepereQueries(self, mappings):
         def GetUpdateQuery(tableName, mappings, pk):
-            query = 'INSERT INTO %s' % tableName
+            query = 'REPLACE INTO %s' % tableName
```

Comment:
Isn't it condrudicting what was changed here?

Reply:
For SQLITE I'm restoring the old behaviour.  REPLACE INTO does not work on any of the versions I've tested... but again, we have no tests.

Reply:
@spicy-sauce is looking into it to check if we can somehow know if insert into is supported, he will update soon.

Reply:
INSERT INTO is supported, I think the issue here is with the REPLACE INTO which is actually an alias to INSERT OR REPLACE INTO.
According to the documentation [1], it should be supported:
The initial "INSERT" keyword can be replaced by "REPLACE" or "INSERT OR action" to specify an alternative constraint conflict resolution algorithm to use during that one INSERT command. For compatibility with MySQL, the parser allows the use of the single keyword REPLACE as an alias for "INSERT OR REPLACE".

I've tested it myself and the two of them work as expected:
sqlite> insert into emp values (1, 'yossi', 'shirizli');
sqlite> select * from emp;
1|yossi|shirizli
sqlite> replace into emp values (1, 'joe', 'schmo');
sqlite>  select * from emp;
1|joe|schmo

[1] https://www.sqlite.org/lang_insert.html

Reply:
Okay - my bad. This is a complete red herring, and I was working off **old** documents that were wrong. The issue is actually somewhere else, so I'm going to close this and blow away the branch.